.travis.yml

```diff
@@ -1,37 +1,57 @@
----
-sudo: false
 language: rust
+sudo: required
```

Comment:
I really wish they'd let this be set per matrix entry... we're going to be slower to schedule because of this.

/me is planning to spend time on https://github.com/kamalmarhubi/containy-thing this week because of this silliness